packages/gatsby/src/utils/worker/child/queries.ts

```diff
@@ -41,4 +45,6 @@ export async function runQueries(queryIds: IGroupedQueryIds): Promise<void> {
   })
 
   await getDataStore().ready()
+
+  saveQueries()
```

Comment:
Do we want to `saveQueries` after each batch or save it once after all queries have been run?

Reply:
Good question! Probably possible both ways and only the question of what we prefer. Maybe we should do it/align it with what we'd do with LMDB in the future cc @vladar

Reply:
I think originally the idea was to sync back once in the end and look for pitfalls (error handling is probably one).

Reply:
Ok, then I'll move it out and run `worker.all.saveQueries()` in the test

Reply:
Done!